src/routes/articles.js

```diff
@@ -107,6 +107,40 @@ const getArticle = async (req, res, next) => {
 	}
 }
 
+// UPDATE ARTICLES ENDPOINT
+
+const updateArticle = async (req, res, next) => {
+	const { id } = req.params // WHERE
+	const { title, article, image, published } = req.body // SET
```

Comment:
meaning of the `// SET` ?

Reply:
it was in the old code, it was meant to make my code understandable. I will take it out since its not needed anymore